{
  "metadata": {
    "timestamp": "2025-02-24T18:15:15Z"
  },
  "data": [
    {
      "cve": {
        "id": "CVE-2018-9456",
        "description": "In sdpu_extract_attr_seq of sdp_utils.cc, there is a possible out of bounds read due to an incorrect bounds check. This could lead to remote denial of service with no additional execution privileges needed. User interaction is not needed for exploitation.",
        "metadata": {
          "published_date": "2024-11-19T23:15:04.100",
          "last_modified_date": "2024-11-22T21:29:34.833",
          "confidence_level": "low",
          "severity": "high"
        },
        "impact": {
          "cisa_kev": false,
          "reported_exploited": false,
          "exploit_maturity": "none",
          "automatable": true
        },
        "counts": {
          "public_exploit_count": 0
        },
        "epss": {
          "score": 0.00046,
          "percentile": 0.1948
        },
        "metrics": [
          {
            "source": "nvd@nist.gov",
            "type": "Primary",
            "cvss_version": "3.1",
            "vector_string": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
            "attack_vector": "NETWORK",
            "base_score": 7.5
          },
          {
            "source": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
            "type": "Secondary",
            "cvss_version": "3.1",
            "vector_string": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
            "attack_vector": "NETWORK",
            "base_score": 7.5
          }
        ],
        "timeline": {
          "references": [],
          "repositories": []
        }
      }
    }
  ]
}